webpack.config.js

```diff
@@ -62,13 +70,42 @@ module.exports = {
           },
         ],
       },
+      {
+        test: /\.pug$/,
+        loader: "pug-plain-loader",
+      },
+      {
+        test: /\.scss$/,
+        use: ["vue-style-loader", "css-loader", "sass-loader"],
+      },
+      {
+        test: /\.sass$/,
+        use: [
+          "vue-style-loader",
+          "css-loader",
+          {
+            loader: "sass-loader",
+            options: {
+              sassOptions: {
+                indentedSyntax: true,
+              },
+            },
+          },
+        ],
+      },
+      {
+        test: /\.less$/,
+        use: ["vue-style-loader", "css-loader", "less-loader"],
+      },
```

Comment:
Forgive my ignorance, I understand that `pug.js` is the templating lib, but `scss`, `sass` and `less`... aren't they ALL CSS preprocessors...? Is there a reason one is not enough but THREE different CSS preprocessor languages are used?

Reply:
Well... I'm not sure whether there is a three-in-one loader, but because they basically all have different language syntax, so maybe three loaders are needed...? I'm not entirely sure, see https://vue-loader.vuejs.org/guide/pre-processors.html#sass

Reply:
Yeah you see my question really is why is *three* different preprocessor languages needed lol.

Reply:
Ahhhhhh, dependencies :)

Reply:
A child dependency uses pug and SCSS for no reason so I have to install it in webpack. I think I no longer need that dep so could maybe remove these now